ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -40,7 +40,13 @@ import RevenueCat
         get { Purchases.proxyURL?.absoluteString }
         set {
             if let value = newValue {
-                guard let proxyURL = URL(string: value) else {
+                var url: URL?
```

Comment:
the good news is that we already had a test for this. 
The bad news is that the test didn't fail 😓  I suppose it's because we weren't testing on iOS 17

Reply:
Oh ugh, that's a bummer